backend/entityservice/views/run/description.py

```diff
@@ -23,21 +25,29 @@ def get(project_id, run_id):
     return RunDescription().dump(run_object)
 
 
+@retry(wait=wait_random_exponential(multiplier=1, max=60), retry=retry_if_exception_type(psycopg2.IntegrityError))
```

Comment:
I still think there should also be a `stop=stop_after...` condition to make sure we not retrying forever.

Reply:
I verified this - you are correct